### PR TITLE
Build svg

### PR DESCRIPTION
**Pull request by nullbitx8**
implement contract methods to build and return the svg image data

### CODE REVIEW

File: contracts/NFcharT.sol [hunk 1]
```diff
@@ -69,40 +79,358 @@ contract NFcharT is ERC721Enumerable, Ownable, ReentrancyGuard {
...
+            // TODO what if twip has 10 leading 0's?
+            Strings.toString(twap), 
+            '</text>'
+        );
+    }
+    
+    /*
+    * @dev Returns SVG of remaining points on graph.
+    */
+    function remainingPointsSVG(uint256[] memory twaps, uint256[7] memory normalizedTwaps, uint16 xInterval) internal pure returns (bytes memory toReturn) {
```
Comment (dallonasnes) [inline contracts/NFcharT.sol]: does this function return anywhere?

**Review by dallonasnes — APPROVED** [review]
Left you a question about return value of `remainingPointsSVG` function. Also you could run the linter if you want but nbd lol. Lgtm!

File: contracts/NFcharT.sol (hunk 1 above)
Comment (nullbitx8) [reply contracts/NFcharT.sol]: Haha good point, it's not clear, but it returns the var 'toReturn' (see end of method signature)
I'm going to change it to explicitly return though, better readability

**Comment by nullbitx8** [comment]
Yea should definitely run the linter. Thanks!

**Pull request merged**